pennylane/templates/qsvt/__init__.py

```diff
@@ -1,45 +0,0 @@
-# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
```

Comment:
Why this part from the parent branch is deleted?

Reply:
We originally thought that this operator would live in this file, but now it seems we might not need it at all.